cloudformation/compute-fleet-hit-substack.cfn.yaml

```diff
@@ -362,12 +362,7 @@ Resources:
               export cookbook_version=${CookbookVersion}
               export chef_version=${ChefVersion}
               export berkshelf_version=${BerkshelfVersion}
-              if [ -f /opt/parallelcluster/.bootstrapped ]; then
-                installed_version=$(cat /opt/parallelcluster/.bootstrapped)
-                if [ "${!parallelcluster_version}" != "${!installed_version}" ]; then
-                  bootstrap_instance
-                fi
-              else
+              if [ ! -f /opt/parallelcluster/.bootstrapped ]; then
```

Comment:
Not sure if we can remove the old code here. I think what would happen here now is: say user is using AMI baked with old version of pcluster, the AMI already has an old version of cookbook on it, without bootstrapping(download and vendor) new version of cookbook onto this old AMI again, we will not actually run new check you might include in the new cookbook version because we will be running the old cookbook on the old AMI.